### PR TITLE
feat: MQTT Command supports viewing connection information 

### LINKED ISSUE #377: RBIP-19: MQTT Command supports viewing connection information
image

### PR DESCRIPTION
**Pull request by PorterZhang2021**
## What's changed and what's your intention?

finish support view connection, but we need a tmp struct build it.

## Checklist

- [x]  I have written the necessary rustdoc comments.
- [x]  I have added the necessary unit tests and integration tests.
- [x]  This PR does not require documentation updates.

## Refer to a related PR or issue link (optional)

close #377

### CODE REVIEW

**Comment by PorterZhang2021** [comment]
cc @socutes

File: src/grpc-clients/src/mqtt/mod.rs [hunk 1]
```diff
@@ -28,6 +28,7 @@ use crate::{retry_sleep_time, retry_times};
 pub enum MqttBrokerService {
     Placement,
     Admin,
+    Connection,
```
Comment (socutes) [inline src/grpc-clients/src/mqtt/mod.rs:31]: Enum "Connection" needs to be removed

File: src/grpc-clients/tests/mqtt/mqtt_connect_test.rs [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    use protocol::broker_mqtt::broker_mqtt_admin::ListConnectionRequest;
+
+    use crate::common::get_mqtt_broker_addr;
+
+    #[tokio::test]
+    async fn test_list_connection() {
+        let client_pool: Arc<ClientPool> = Arc::new(ClientPool::new(3));
+        let addrs = vec![get_mqtt_broker_addr()];
+
+        match mqtt_broker_list_connection(client_pool, addrs, ListConnectionRequest {}).await {
```
Comment (socutes) [inline src/grpc-clients/tests/mqtt/mqtt_connect_test.rs:30]: @PorterZhang2021 mqtt_broker_list_connection(client_pool, addrs, ListConnectionRequest {}).await.unwrap();

**Review by socutes — APPROVED** [review]
LGTM

**Pull request merged**